pom.xml

```diff
@@ -14,26 +15,25 @@
     <licenses>
         <license>
             <name>MIT License</name>
-            <url>http://www.opensource.org/licenses/mit-license.php</url>
+            <url>https://opensource.org/licenses/MIT</url>
         </license>
     </licenses>
     
     <properties>
-      <jenkins.version>1.609.1</jenkins.version>
-      <java.level>6</java.level>
+      <jenkins.version>1.642.3</jenkins.version>
```

Comment:
Not sure why you would need it, but OK since you are a maintainer.
1.625 would be enough for try-with-resources